nautobot/extras/utils.py

```diff
@@ -80,6 +81,45 @@ def get_choices(self):
         return [(f"{ct.app_label}.{ct.model}", ct.pk) for ct in ContentType.objects.filter(self.get_query())]
 
 
+@deconstructible
+class ModelSubclassesQuery:
+    """
+    Helper class to get ContentType models that are subclasses of self.klass
+    """
+
+    # TODO: Make ModelSubclassesQuery reusable by adding a klass arg to ___init__.
+    # Instead of geting the subclasses of only PrimaryModel, the subclasses
+    # of any class provided can be generated e.g ModelSubclassesQuery(RelationshipModel)
```

Comment:
If circular imports prove to be an unsolvable problem here, one fallback possibility might be to pass in the name (e.g. `"nautobot.core.models.generics.PrimaryModel"`) and use `import_object` to load it as needed?